lib/babe/errors.go

```diff
@@ -64,27 +64,27 @@ var (
-	errInvalidResult                   = errors.New("invalid error value")
-	errNoEpochData                     = errors.New("no epoch data found for upcoming epoch")
-	errFirstBlockTimeout               = errors.New("timed out waiting for first block")
-	errChannelClosed                   = errors.New("block notifier channel was closed")
-	errOverPrimarySlotThreshold        = errors.New("cannot claim slot, over primary threshold")
-	errNotOurTurnToPropose             = errors.New("cannot claim slot, not our turn to propose a block")
-	errSecondarySlotProductionDisabled = errors.New("cannot claim slot, secondary slot production is disabled")
-	errNoConfigData                    = errors.New("cannot find ConfigData for epoch")
-	errGetEpochData                    = errors.New("get epochData error")
-	errFailedFinalisation              = errors.New("failed to check finalisation")
-	errMissingDigest                   = errors.New("chain head missing digest")
-	errSetFirstSlot                    = errors.New("set first slot error")
-	errGetEpoch                        = errors.New("get epoch error")
-	errSkipVerify                      = errors.New("skipVerify error")
-	errMissingDigestItems              = errors.New("block header is missing digest items")
-	errDescendant                      = errors.New("descendant err")
-	errServicePaused                   = errors.New("service paused")
+	errNilBlockImportHandler    = errors.New("cannot have nil BlockImportHandler")
+	errNilEpochState            = errors.New("cannot have nil EpochState")
+	errNilStorageState          = errors.New("storage state is nil")
+	errNilParentHeader          = errors.New("parent header is nil")
+	errInvalidResult            = errors.New("invalid error value")
+	errNoEpochData              = errors.New("no epoch data found for upcoming epoch")
+	errFirstBlockTimeout        = errors.New("timed out waiting for first block")
+	errChannelClosed            = errors.New("block notifier channel was closed")
+	errOverPrimarySlotThreshold = errors.New("cannot claim slot, over primary threshold")
+	errNotOurTurnToPropose      = errors.New("cannot claim slot, not our turn to propose a block")
+	errNoConfigData             = errors.New("cannot find ConfigData for epoch")
+	errGetEpochData             = errors.New("get epochData error")
+	errFailedFinalisation       = errors.New("failed to check finalisation")
+	errMissingDigest            = errors.New("chain head missing digest")
+	errSetFirstSlot             = errors.New("set first slot error")
+	errGetEpoch                 = errors.New("get epoch error")
+	errSkipVerify               = errors.New("skipVerify error")
+	errMissingDigestItems       = errors.New("block header is missing digest items")
+	errDescendant               = errors.New("descendant err")
+	errServicePaused            = errors.New("service paused")
+	errInvalidSlotTechnique     = errors.New("invalid slot claiming technique")
```

Comment:
I hate the Go formatter for introducing useless deltas like that one... come we just changed 2 lines!